pymc_marketing/mmm/transformers.py

```diff
@@ -1,7 +1,61 @@
 import pytensor.tensor as pt
+from pytensor.tensor.random.utils import params_broadcast_shapes
 
 
-def geometric_adstock(x, alpha: float = 0.0, l_max: int = 12, normalize: bool = False):
+def batched_convolution(x, w, axis: int = 0):
```

Comment:
This looks like it took 5 minutes to write...

Reply:
What do you mean?

Btw I didn't write this code, it's picked directly from @lucianopaz PR